taxi/views.py

```diff
@@ -60,12 +61,28 @@ class CarListView(LoginRequiredMixin, generic.ListView):
 
 class CarDetailView(LoginRequiredMixin, generic.DetailView):
```

Comment:
It’s ok, but consider separating the logic of adding and removing drivers into distinct views ( for example, `CarAssignDriverView` and `CarRemoveDriverView`) to follow the **Single Responsibility Principle**, rather than combining it in `CarDetailView`.